test/cypress/integration/request-psp-test-account/submit-request-for-psp-test-account.cy.js

```diff
@@ -1,29 +1,34 @@
 const userStubs = require('../../stubs/user-stubs')
 const serviceStubs = require('../../stubs/service-stubs')
-const zendeskStubs = require('../../stubs/zendesk-stubs')
+const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
+const tokenStubs = require('../../stubs/token-stubs')
 
 describe('Request PSP test account: submit request', () => {
   const userExternalId = 'cd0fa54cf3b7408a80ae2f1b93e7c16e'
-  const gatewayAccountId = 42
+  const sandboxGatewayAccountId = 42
   const serviceExternalId = 'afe452323dd04d1898672bfaba25e3a6'
   const requestStripeTestAccountUrl = `/service/${serviceExternalId}/request-stripe-test-account`
+  const stripeGatewayAccountExternalId = 'a-stripe-gw-external-id'
 
   const setupStubs = (pspTestAccountStageFirstResponse, pspTestAccountStageSecondResponse) => {
     cy.task('setupStubs', [
       userStubs.getUserSuccessRespondDifferentlySecondTime(userExternalId,
         {
-          gatewayAccountId,
+          gatewayAccountId: sandboxGatewayAccountId,
           serviceExternalId,
           pspTestAccountStage: pspTestAccountStageFirstResponse
         },
         {
-          gatewayAccountId,
+          gatewayAccountId: sandboxGatewayAccountId,
           serviceExternalId,
           pspTestAccountStage: pspTestAccountStageSecondResponse
         }
       ),
-      zendeskStubs.createTicketSuccess(),
-      serviceStubs.patchUpdateServicePspTestAccountStage({ serviceExternalId, gatewayAccountId, pspTestAccountStage: 'REQUEST_SUBMITTED' })
+      gatewayAccountStubs.getAccountByServiceIdAndAccountType(serviceExternalId, { gateway_account_id: sandboxGatewayAccountId }),
+      gatewayAccountStubs.requestStripeTestAccount(serviceExternalId, { gateway_account_external_id: stripeGatewayAccountExternalId }),
+      gatewayAccountStubs.addGatewayAccountsToService(serviceExternalId),
+      serviceStubs.patchUpdateServicePspTestAccountStage({ serviceExternalId, gatewayAccountId: sandboxGatewayAccountId, pspTestAccountStage: 'REQUEST_SUBMITTED' }),
+      tokenStubs.revokeTokensForAccount(sandboxGatewayAccountId)
```

Comment:
by including this stub, does this verify that the method is called in the test?

Reply:
It doesn't, but node isn't my expertise and I can't see anywhere else this sort of verification gets done in the selfservice codebase.

Reply:
I think in the unit tests there's `sinon.assert.called` and `sinon.assert.calledWith`, but I don't know if there's an equivalent in cypress or even if that's sensible. But we probably do want to avoid a situation where a subsequent change stops the tokens from being revoked.

Reply:
Yeah given there isn't an equivalent in cypress tests I'd like to ignore this for now. Part of the reason being this PR has been open for a while now, and the more it's open the more merge hell I'll have to deal with, so if we deal with this let it be another PR.